mitodl/open-discussions
# Use Django models for moderator, contributor serializers & views

Based off of PR #1500

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes #1501
Related to #1466

#### What's this PR do?
- Changes serializers and API views for moderators and contributors to query django instead of reddit.

#### How should this be manually tested?
Members, Contributors:
 - Go to the 'Members' and 'Contributors' tabs for a channel you moderate.  The list of users should still work, and you should still be able to add and remove the same users as you would on master (for moderators, you should be able to remove users added after you became a moderator but not before).

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)

#### What GIF best describes this PR or how it makes you feel?
(Optional)
